docs/api.md

```diff
@@ -28,11 +28,15 @@ The Loki server has the following API endpoints (_Note:_ Authentication is out o
 
   - `query`: a logQL query
   - `limit`: max number of entries to return
-  - `start`: the start time for the query, as a nanosecond Unix epoch (nanoseconds since 1970)
-  - `end`: the end time for the query, as a nanosecond Unix epoch (nanoseconds since 1970)
+  - `start`: the start time for the query, as a nanosecond Unix epoch (nanoseconds since 1970). Default is always one hour ago.
+  - `end`: the end time for the query, as a nanosecond Unix epoch (nanoseconds since 1970).
   - `direction`: `forward` or `backward`, useful when specifying a limit
```

Comment:
Default is backward.

Reply:
Also: should document explicitly how forward/backward changes behaviour.  i.e. if you do 'backward' then you'd think that logs are shown from 'end' time backwards, until 'limit' is reached.  And therefore you'd issue a new query, with 'end' = last 'ts' time, to continue retrieving logs.

But what's not clear is how this works if the "ts" values of incoming logs were not monotonically increasing (given that each process inserting logs can apply its own "ts")

I can see that loki is loosely modelled on AWS CloudWatch logs, but the [GetLogEvents](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetLogEvents.html) call has an opaque 'token' which embeds the exact position in the stream, so you can continue to get more logs.

Reply:
Hmm. I'm not sure how the behaviour works. I've just used forward as it fits my use case better. I would rather not add that as I'm not exactly sure.